ZenPacks/zenoss/OpenStackInfrastructure/hostmap.py

```diff
@@ -261,7 +265,7 @@ def get_hostname_for_hostid(self, hostid):
             raise Exception("perform_mapping must be called before get_hostname_for_hostid")
 
         if not hostid.startswith("host-"):
-            raise Exception("get_hostname_for_hostid must be supplied with a valid hostid")
+            raise InvalidHostIdException("get_hostname_for_hostid must be supplied with a valid hostid")
 
```

Comment:
I didn't modify that message in hostmap. I don't show that message in the modeler though.